db/document.go

```diff
@@ -66,18 +77,52 @@ type syncData struct {
 // In its JSON form, the body's properties are at top-level while the syncData is in a special
 // "_sync" property.
 type document struct {
-	syncData
-	body Body
-	ID   string `json:"-"`
-	Cas  uint64
+	syncData        // Sync metadata
+	_body    Body   // Marshalled document body.  Unmarshalled lazily - should be accessed using Body()
+	rawBody  []byte // Raw document body, as retrieved from the bucket
+	ID       string `json:"-"` // Doc id
```

Comment:
Can you add to the comment and mention why the ID field needs to be omitted?  I think it would be useful for someone who came along and wanted to remove the `-` tag.